fabric/src/main/resources/fabric.mod.json

```diff
@@ -0,0 +1,24 @@
+{
+  "schemaVersion": 1,
+  "id": "nuvotifier",
+  "version": "${internalVersion}",
+
+  "name": "NuVotifier",
+  "description": "Safe, smart, and secure Votifier server mod",
+  "authors": [
+    "Drex"
```

Comment:
I'll let this slide for now but I'll change this to "NuVotifier Authors" in all the ports